package.json

```diff
@@ -7,7 +7,7 @@
     "test": "test"
   },
   "scripts": {
-    "test": "npm run lint && mocha --recursive test",
+    "test": "npm run lint && mocha",
```

Comment:
I think we still want to limit mocha to the `test` folder

Reply:
Defaults to test directory and I don't see the point in the --recursive option since there are not any subdirectories. [source](https://github.com/mochajs/mocha/blob/master/lib/cli/run.js#L238-L242)

Sticks with syntax of three other repos: [runner](https://github.com/mongodb-js/runner/blob/master/package.json#L8), [is-mongodb-running](https://github.com/mongodb-js/is-mongodb-running/blob/master/package.json#L7), & [version-manager](https://github.com/mongodb-js/version-manager/blob/master/package.json#L15)